libraries/mbed/hal/can_api.h

```diff
@@ -37,7 +37,8 @@ typedef enum {
     IRQ_PASSIVE,
     IRQ_ARB,
     IRQ_BUS,
-    IRQ_READY
+    IRQ_READY,
+    IRQ_RX1
```

Comment:
You're right it is specific to STM32. We would have liked to get IRQ_RX0 and IRQ_RX1, but I did not want to change existing code for other partners. 
I only added a new define. 
Some of the current enum are not used in our board. Isn't IRQ_ARB specific to another module ? It shouldn't be an issue as long as it is an enum and everyone implements a 'default' case.